Source/Xamarin/Prism.Forms.Tests/Services/PageDialogServiceFixture.cs

```diff
@@ -20,68 +20,273 @@ public PageDialogServiceFixture()
         [Fact]
         public async Task DisplayActionSheetNoButtons_ShouldThrowException()
         {
-            var service = new PageDialogServiceMock("cancel", _applicationProvider);
-            var argumentException = await Assert.ThrowsAsync<ArgumentException>(() => service.DisplayActionSheetAsync(null, null));
-            Assert.Equal(typeof(ArgumentException), argumentException.GetType());
+            var service = new PageDialogServiceMock( "cancel", _applicationProvider );
+            var argumentException = await Assert.ThrowsAsync<ArgumentException>( () => service.DisplayActionSheetAsync( null, null ) );
+            Assert.Equal( typeof( ArgumentException ), argumentException.GetType() );
+        }
+
+        #region Tests using Action
+
+        private async Task DisplayActionSheet_PressButton_UsingAction( string text )
```

Comment:
Please place all private helper methods/functionality at the bottom of the class, test methods are more important and should come first.